tendrl-node-agent.spec

```diff
@@ -24,6 +24,7 @@ Requires: collectd
 Requires: python-jinja2
 Requires: tendrl-commons
 Requires: hwinfo 
+Requires: pytz
```

Comment:
@TimothyAsir is this really required? i never see this dependency when i test socket